package.json

```diff
@@ -30,26 +32,45 @@
   "scripts": {
     "start": "stripes serve",
     "lint": "eslint .",
-    "test": "echo 'placeholder. no tests implemented'"
+    "test": "stripes test karma"
   },
   "devDependencies": {
-    "@folio/eslint-config-stripes": "^5.1.0",
-    "@folio/stripes": "^4.0.0",
     "@folio/stripes-cli": "^1.3.1",
+    "localforage": "^1.5.6",
+    "@bigtest/convergence": "^1.1.1",
+    "@bigtest/interactor": "^0.9.1",
+    "@bigtest/mocha": "^0.5.2",
+    "@bigtest/react": "^0.1.2",
+    "@folio/eslint-config-stripes": "^5.1.0",
+    "regenerator-runtime": "^0.13.3",
     "babel-eslint": "^10.0.3",
+    "chai": "^4.2.0",
     "eslint": "^6.2.1",
-    "react": "^16.5.0",
-    "react-dom": "^16.5.0",
-    "react-intl": "^4.5.3"
+    "mocha": "^8.0.1",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-intl": "^4.5.3",
+    "react-router": "^5.2.0",
+    "react-router-dom": "^5.2.0",
+    "sinon": "^7.2.2"
   },
   "dependencies": {
+    "@folio/stripes-components": "~8.0.0",
+    "@folio/stripes-connect": "~6.0.0",
+    "core-js": "^3.6.4",
+    "@folio/stripes": "^4.0.0",
+    "@folio/stripes-core": "^5.0.0",
```

Comment:
Not quite right; the deps should include `@folio/stripes` only. If you need access to un-exported things from, e.g. stripes-components for interactors in testing, then you can include those packages as dev-deps. At the dependencies level, however, the only stripes import should be `@folio/stripes`.

Reply:
Also, if you're depending on react-router v5 then your stripes dep should be `^5.0.0`. In fact, the components and core deps you have here are incompatible with each other, which is exactly why we created `@folio/stripes` as a single entity that handles aligning all those deps for you.